lib/query.js

```diff
@@ -72,4 +116,112 @@ export const getDenominator = (node) => {
     } else {
         return null
     }
-}
\ No newline at end of file
+}
+
+// TODO: handle multivariable polynomials
+// Get degree of a polynomial term
+// e.g. 6x^2 -> 2
+export const getPolyDegree = (node) => {
+    if (isNumber(node)) {
+        return build.number(0)
+    } else if (isIdentifier(node) || isPolynomial(node)){
+        return build.number(1)
+    } else if (isPow(node)) {
+        return node.args[1]
+    } else if (isMul(node)){
+        return getPolyDegree(node.args[1])
```

Comment:
Good point.  We'd have to convert each degree to a number before calling `Math.max`.